backend/src/govsg/models/govsg-verification.ts

```diff
@@ -0,0 +1,29 @@
+import {
+  Column,
+  DataType,
+  ForeignKey,
+  Model,
+  Table,
+} from 'sequelize-typescript'
+import { GovsgMessage } from './govsg-message'
+
+@Table({ tableName: 'govsg_verification', underscored: true, timestamps: true })
+export class GovsgVerification extends Model<GovsgVerification> {
```

Comment:
The passcode - govsg_message relation is 1-1 right? Do we actually need another model for this? Or would new fields on govsg_message model suffice?

Reply:
Not exactly because `govsgMessageId` comes from the initial message with `params` and `passcodeCreationWamid` will be populated by the 2nd message with the quick-reply button.

As for allowing users to see non-send/delivery of the `passcodeCreationWamid`, I would want to think of this later.

Reply:
And it won't be easy adding passcode as a column because:

- Not all messages will have `passcode` and `passcodeCreationWamid`.
- Having a separate relation is cleaner - in this new relation, we can establish everything as non-null because all values can be defined before creation. This is not the case when we add columns to `govsg_messages`. Let me make a commit to set `allowNull: false`.